src/main/java/frc/robot/commands/RotatorToPosition.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.subsystems.RotatorSubsystem;
+import frc.robot.subsystems.TelescopeSubsystem;
+
+public class RotatorToPosition extends CommandBase {
+    private final RotatorSubsystem rotatorSubsystem;
+    private final TelescopeSubsystem telescope;
+    private final double setpoint;
+    private final ResetTelescope resetTelescope;
+    private boolean resettingElevator;
+    public RotatorToPosition(RotatorSubsystem rotatorSubsystem, TelescopeSubsystem telescope, double setpoint)
+    {
+        this.rotatorSubsystem = rotatorSubsystem;
+        this.setpoint = setpoint;
+        this.telescope = telescope;
+        addRequirements(rotatorSubsystem);
+
+        resetTelescope = new ResetTelescope(telescope);
+    }
+    @Override
+    public void initialize() {
+        if (telescope.getCurrentPosition() > 2048 && Math.abs(telescope.getCurrentPosition() - setpoint) > 4096){
```

Comment:
If would be nice to not expose telescope position in ticks. Also, aren't extension ticks negative right now? (And if they are, consider inverting the motor.)

Consider percentage, or inches instead of ticks.

Reply:
After some investigation, inverting the motor doesnt actually invert the encoder, for some stupid reason. I'm not sure how to invert the encoder itself on the motor. I'll do some research.

I implemented percentage based on AutoPrime's code.